ansible/roles_ocp_workloads/ocp4_workload_migration_training/tasks/workload.yml

```diff
@@ -0,0 +1,131 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Remove default Catalogs from OperaturHub
+  when: ocp4_workload_migration_training_disable_default_catalogs | bool
+  k8s:
+    state: present
+    definition: "{{ lookup('template', './templates/operatorhub.yaml.j2' ) | from_yaml }}"
+  vars:
+    __ocp4_workload_migration_training_disable_operatorhub: true
+
+- name: Create openshift-migration namespace
+  k8s:
+    state: present
+    definition: "{{ lookup('template', './templates/namespace.yaml.j2' ) | from_yaml }}"
+
+- name: Create Catalogsource for use with catalog snapshot
+  when: ocp4_workload_migration_training_use_catalog_snapshot | bool
+  k8s:
+    state: present
+    definition: "{{ lookup('template', './templates/catalogsource.yaml.j2' ) | from_yaml }}"
+
+- name: Install Migration Toolkit for Containers
+  when: ocp4_workload_migration_training_install_mtc | bool
+  block:
+  - name: Create Migration Toolkit for Containers subscription
+    k8s:
+      state: present
+      definition: "{{ lookup('template', item ) | from_yaml }}"
+    loop:
+    - ./templates/operatorgroup.yaml.j2
+    - ./templates/subscription.yaml.j2
+
+  - name: Wait until InstallPlan is created
+    k8s_info:
+      api_version: operators.coreos.com/v1alpha1
+      kind: InstallPlan
+      namespace: openshift-migration
+    register: r_install_plans
+    vars:
+      _query: >-
```

Comment:
You use `_query` here and `query` in the subsequent task. Was this intentional or can we keep the var names the same for consistency since they live in the task? Also, should it be `__`? I am confused on our standard at this point sigh.

Reply:
Done